### ISSUE TITLE
Start menu all apps doesn't work the first time after login

### ISSUE THREAD

**Issue by Insprill** [ramensoftware/windhawk-mods#572]
After signing in after a reboot, or after logging out, the first time the start menu is opened, it opens to the default page instead of to the all apps page. All subsequent openings are correct.

Author: @m417z

**Comment by m417z** [comment]
It may happen if code is injected with a delay. StartMenuExperienceHost.exe is started by svchost.exe. What might help is adding svchost.exe to the process inclusion list in the advanced settings. It will make it more likely for the mod to be loaded at an early stage. Try it and let me know if it helps.

![image](https://github.com/ramensoftware/windhawk-mods/assets/4129781/ddfd282f-c5d2-4d85-a130-83fb1500f3ae)

**Comment by Insprill** [comment]
I assume you mean the `Custom process inclusion list` advanced setting for the mod? I don't see the exact UI you showed there. Adding `svchost.exe` to the aforementioned list did not help. After saving and restarting, the issue persists and is still reproducible when logging out and back in.

I realize I didn't add my system info in the original issue; I'm running Windows 11 23H2 build 22631.3296, Windhawk v1.4.1, and Show all apps v1.0.2

**Comment by m417z** [comment]
> I assume you mean the `Custom process inclusion list` advanced setting for the mod?

No, that's not what I meant and that's not the correct place for svchost.exe, please remove it.

Open Windhawk, go to Settings, then Advanced settings, then scroll down and click on More advanced settings.

**Comment by Insprill** [comment]
Apologies, I missed the "More advanced settings" button. That indeed fixes the issue!
